packages/lit-dev-content/site/docs/v3/components/properties.md

```diff
@@ -129,10 +129,13 @@ class MyElement extends LitElement {
 }
 ```
 
-In **JavaScript**, you **must not use class fields** when declaring reactive properties. Instead, properties must be initialized in the element constructor:
+In **JavaScript**, you **must not use class fields** when declaring reactive properties. Instead, properties must be either initialized in the element constructor or set with the `defaultValue` option:
```

Comment:
While you're here "Instead" reads a little unclearly to me. I think saying "to initialize properties" would help somewhere. Also, "JavaScript" now has decorators via Babel at least, so maybe we update that phrase too.

Maybe:

```suggestion
If you are declaring reactive properties with the static properties class field, and not using decorators, you **must not use class fields** for those properties. Class fields will overwrite the properties in the static properties class field, disabling reactivity. To initialize a reactive property, do so in the element constructor or set the `defaultValue` option:
```